tasklist/src/pages/modalAddTask/modalAddTask.ts

```diff
@@ -83,21 +89,43 @@ export class ModalAddTask {
     );
   }
 
+  populateTask() {
+    // Populate the interface with the task information
+  }
+
   dismiss() {
     this.viewCtrl.dismiss();
   }
 
   createForm() {
-    console.log(this.task);
-    this.taskService.createTask(this.task, this.project).subscribe(
+    let handler;
+    if( this.task.id ) {
```

Comment:
I have an error when i launch the app :
```
[13:41:14]  typescript: src/pages/modalAddTask/modalAddTask.ts, line: 102 
            Property 'id' does not exist on type '{}'. 

     L101:  let handler;
     L102:  if( this.task.id ) {
     L103:    handler = this.editTask(this.task, this.project);
```

If i replace `task = {};` by `task: any;` the problem seem solve on my side.